docs/assets/css/anchorjs.css

```diff
@@ -5,6 +5,7 @@
 .anchorjs-link {
     color: inherit !important;
     text-decoration: none !important; /* do not underline */
+    margin-left: -1.6em; /* inorder to avoid overlap with the icon */
```

Comment:
A little correction, it should be "-1.6 rem"

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient